src/coreComponents/physicsSolvers/SolverBase.cpp

```diff
@@ -52,7 +52,7 @@ SolverBase::SolverBase( string const & name,
   this->setTimesteppingBehavior( ExecutableGroup::TimesteppingBehavior::DeterminesTimeStepSize );
 
   registerWrapper( viewKeyStruct::cflFactorString(), &m_cflFactor ).
-    setApplyDefaultValue( 0.5 ).
+    setApplyDefaultValue( -1.0 ).
     setInputFlag( InputFlags::OPTIONAL ).
     setDescription( "Factor to apply to the `CFL condition <http://en.wikipedia.org/wiki/Courant-Friedrichs-Lewy_condition>`_"
                     " when calculating the maximum allowable time step. Values should be in the interval (0,1] " );
```

Comment:
for FIM, does it still have to be <=1? What if one wants to push CFL to 10, 100, etc?

Reply:
We can push it above 1 for sure. For instance to test C1-ppu and IHU I ran cases at 50 and 320 as reported in articles. `0.5` is more as an obvious default than a recommendation

Reply:
Ok great, maybe change the description to avoid confusion?

Reply:
Ok, I will the introduce a new _CFL_ for flow, as I was, during the development, living off the `m_cflFactor` that is being used in hydrofrac to spare me some proliferation of CFLs. At this point I think it is legit to make difference between those two thing.